### DISCUSSION TITLE
Feature listed in Changelog missing from docker image

### DISCUSSION THREAD

**Discussion by plerohellec** [plausible/analytics#2075] [Self-Hosted Support] [answered]
The feature I was looking for is listed in the [changelog](https://github.com/plausible/analytics/blob/master/CHANGELOG.md) for version 1.4.0: "Choice of metric for main-graph both in UI and API (visitors, pageviews, bounce_rate, visit_duration) [1364](https://github.com/plausible/analytics/pull/1364)". I can see the feature on the [demo](https://plausible.io/plausible.io) page.
But after installing plausible/analytics:v1.4.4 from my Dockerfile ("FROM plausible/analytics:v1.4.4"), the feature is not available: the labels above the main chart "Total Visitors" and "Unique Pageviews" are still just labels when I was expecting links. I also tried the `latest` docker image.

I would not rule out that I've done something wrong and the version I'm running is not the one I believe it is but `docker images` shows `plausible/analytics        v1.4.4          f2bfe5a49b44   6 months ago    274MB`. 

I do see that even though the feature is listed in the changelog, it's not in the [github releases page](https://github.com/plausible/analytics/releases) for 1.4.0. Is it possible that the changelog  lists changes for the Plausible cloud version but not all of those changes are in the docker images? Or is there a configuration option that needs to be turned on to be able to change the content of the main graph to pageviews? Thank you!

**Comment by metmarkosaric** [comment, marked as answer]
thanks for reporting @plerohellec! this is a relatively new feature and still unreleased in self-hosted. i've now fixed the changelog
